### PR TITLE
data migration issues

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/features/app/app-slice.js [hunk 1]
```diff
@@ -145,10 +145,12 @@ export const appOperations = {
     return;
   },
   initialize: () => async (dispatch, getState) => {
-    SplashScreen.hide();
-
     const wallet = Wallet.getInstance();
 
+    wallet.eventManager.on(WalletEvents.migrated, () => {
+      SplashScreen.hide();
```
Comment (mike-parkhill) [inline src/features/app/app-slice.js:151]: does the splashscreen get closed elsewhere when no migration is run?

Comment (maycon-mello) [reply src/features/app/app-slice.js:151]: migration event will happen every time we open the wallet, to check if the current version is the latest. It means the app is ready to be used

Comment (maycon-mello) [reply src/features/app/app-slice.js:151]: if a migration is required it will just take more time to remove the splash screen, which is expected

**Comment by maycon-mello** [comment]
We need to merge https://github.com/docknetwork/react-native-sdk/pull/40 before merging this one

**Pull request merged**